packages/conventional-changelog-conventionalcommits/writer-opts.js

```diff
@@ -175,7 +175,7 @@ function getWriterOpts (config) {
 
 // merge user set configuration with default configuration.
 function defaultConfig (config) {
-  config = config || {}
+  config = Object.assign({}, config)
```

Comment:
can you use spread here?

```suggestion
function defaultConfig (originalConfig) {
  const config = { ...originalConfig }
```